### ISSUE TITLE
Migration tool fails from owncloud 8.2.11 installed via apt on debian 7

### ISSUE THREAD

**Issue by dirks** [nextcloud/server#5317]
### Steps to reproduce
1. Install owncloud on debian 7 from `deb http://download.owncloud.org/download/repositories/8.2/Debian_8.0 /`
2. follow the instructions on the migration tool listed ad https://nextcloud.com/migration/ 
 
### Expected behaviour
Migration to Nextcloud 9.0.58 should succeed.

### Actual behaviour
- updater folder is missing, but 
  ```
  mkdir updater
  mv index.php1 updater/index.php
  chown -R www-data:www-data updater
  ```
- updater complains about `assets` folder, but it was empty anyway `rmdir assets`
- updater fails on check for writer permissions on `/var/www/owncloud/updater/.../uer.ini` and many more files it probably expects below the updater directory. Permissions itself should be fine as
  - updater folder www-data:www-data 755 
  - updater/index.php  www-data:www-data 644

### Server configuration

**Operating system**: Debian 8.8 jessie

**Web server:** apache 2.4.10-10+deb8u8

**Database:** mysql-server 5.5.55-0+deb7u1 on debian 7.11 wheezy

**PHP version:** 5.6.30+dfsg-0+deb8u1

**Nextcloud version:** (see Nextcloud admin page) none yet

**Updated from an older Nextcloud/ownCloud or fresh install:** trying to migrate from owncloud 8.2.11 installed via `deb http://download.owncloud.org/download/repositories/8.2/Debian_8.0 /`

**Where did you install Nextcloud from:** not yet

**Comment by enoch85** [comment]
@dirks The thing is that the migration tool doesn't replace the deb stuff - it replaces the actual ownCloud dir. So if you installed ownCloud with deb packages, make sure to disable them after the migration is done. 

Second, in some cases ownCloud packages php dependencies as well, it depends on which deb you chose to install from start. In case PHPX depends on ownCloud you need to reinstall that as well.

Third, the updater requires www-data permissions on the root and subfolders so `sudo chown www-data:www-data -R /var/www/owncloud should solve the issue.

If you want to set strong permissions again feel free to use this script from the Nextcloud VM: https://github.com/nextcloud/vm/blob/master/static/setup_secure_permissions_nextcloud.sh

If you need personal support migrating from ownCloud to Nextcloud there are alternatives. One of them can be found [here](https://shop.techandme.se/index.php/product/migrate-from-owncloud-to-nextcloud/).

**Comment by dirks** [comment]
@enoch85 thanks for the pointers. Point 1 and 2 are clear and I took care of that. Point 3 was not, as the error messages only mentions files below ./updater. I got the hint on IRC that the owncloud installation might be too old, so I suspected a bug in the documentation, which mentions owncloud 8.2.11 to nextcloud 9 as supported upgrade path.

I continued the migration manually (8.2 -> 9 -> 10 -> 11 -> 12). As this was a test for the production system only, I will try the migration tool again on production (after another db backup and vm snapshot) and report back. If setting owner and group to www-data for the whole owncloud/nextcloud directory tree solves the issue the error message or the documentation of the migration tool could be improved slightly.
